### ISSUE TITLE
Use tabs for indentation instead of spaces

### ISSUE THREAD

**Issue by hrivu21** [chipsalliance/verible#1859]
Currently, on formatting, all the indentations are in spaces; even tabs are converted to spaces on running the formatter.

Can a flag be introduced to choose between tabs and spaces for indentation?

**Comment by tgorochowik** [comment]
Indeed, currently the only configuration is for the number of spaces to be used for indentation (` --indentation_spaces`

Adding a configuration for tabs should be doable.
